packages/salesforcedx-vscode-apex/package.json

```
@@ -193,104 +193,104 @@
     "menus": {
       "commandPalette": [
         {
-          "command": "sfdx.anon.apex.execute.selection",
-          "when": "sfdx:project_opened && editorHasSelection && sfdx:has_default_username"
+          "command": "sf.anon.apex.execute.selection",
+          "when": "sf:project_opened && editorHasSelection && sf:has_default_username"
```

Should we replace `default_username` with `target_org` given that CLI team has already made this change?  (If yes, the change needs to be made in all places in this file and other files where it is present.)

```suggestion
          "when": "sf:project_opened && editorHasSelection && sf:has_target_org"
```